tests/yam/validate/validate_user.pm

```diff
@@ -0,0 +1,33 @@
+# SUSE's openQA tests
+#
+# Copyright 2023 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: Validate alp user with password
+# Maintainer: QE YaST and Migration (QE Yam) <qe-yam at suse de>
+
+use base "consoletest";
+use strict;
+use warnings;
+use testapi;
+use serial_terminal qw(select_serial_terminal);
+
+sub run {
+    my $user = "test";
```

Comment:
user should be added properly by agama, we could just login to check that the password is fine.

Reply:
The user is root after we call select_serial_terminal(), if you switch root to other user, no need input any password.
That's the reason for current logic, root-> test->bernhard.